src/components/MultipleChoice/BaseMultipleChoice.tsx

```diff
@@ -48,6 +50,8 @@ export const BaseMultipleChoice: FC<BaseMultipleChoiceProps> = ({
 
 		// onKeyDown will check which key is pressed and conditionally select/deselect item or give focus to item
 		const onKeyDown = (e: KeyboardEvent) => {
+			keysPressedMap[e.key] = e.type === 'keydown'
```

Comment:
Would `e.type` ever not be `keydown` if the event listener is activated onKeyDown? Should this just be `true` instead?

Reply:
Yes it should be. In an earlier implementation this value was being set to `false` on keyup